src/app/shared/appDownloadHelper.js

```diff
@@ -1,22 +1,15 @@
-const {
-  SERVICE_URL,
-  SERVICE_DOMAIN,
-  ANDROID_APP_ID,
-} = require("../../lib/config");
+const { SERVICE_URL } = require("../../lib/config");
 const PHONE_TYPES = require("../../constants/phone-types");
+const path = require("path");
 
 // Returns the url of the app redirect route for the requested phone type. For Android phones the url is wrapped in an
 // intent so the app is launched if it is already installed on the device.
 function getAppStoreRedirectUrl(phoneType) {
-  const browserUrl = SERVICE_URL + "/ipv/app-redirect/" + phoneType;
-
-  if (phoneType === PHONE_TYPES.IPHONE) {
-    return browserUrl;
-  } else if (phoneType === PHONE_TYPES.ANDROID) {
-    return `intent://${SERVICE_DOMAIN}/ipv/app-redirect/#Intent;scheme=https;package=${ANDROID_APP_ID};S.browser_fallback_url=${encodeURIComponent(browserUrl)};end`;
+  if (phoneType !== PHONE_TYPES.IPHONE && phoneType !== PHONE_TYPES.ANDROID) {
+    throw new Error(`Unrecognised phone type: ${phoneType}`);
   }
 
-  throw new Error(`Unrecognised phone type: ${phoneType}`);
+  return path.join(SERVICE_URL || "/", "ipv", "app-redirect", phoneType);
```

Comment:
Careful - `path` is for file paths, not URLs.

On Windows this will produce `\ipv\app-redirect\android` (with backslashes, not forward!)